projects/tuist/fixtures/ios_app_with_framework_and_disabled_resources/Tuist/Config.swift

```diff
@@ -0,0 +1,8 @@
+import ProjectDescription
+
+let config = Config(
+  generationOptions: [
+    .disableBundleAccessors,
```

Comment:
thanks a lot for adding this fixture! It's very much appreciated.

However, I think it'd be great if you also ran some acceptance test for this to ensure this fixture is compilable. You can also assert that those derived files are not created.

You can add the acceptance test [here](https://github.com/tuist/tuist/blob/main/projects/tuist/features/generate-8.feature). There is no assertion at the moment for files not existing, but you can just more-or-less copy [this](https://github.com/tuist/tuist/blob/main/projects/tuist/features/step_definitions/generate.rb#L126) and negate it.

You can read more about acceptance tests [here](https://docs.tuist.io/contributors/testing-strategy)